package.json

```diff
@@ -51,11 +51,18 @@
       "title": "Sourcegraph extension configuration",
       "properties": {
         "sourcegraph.url": {
-          "type": [
-            "string"
-          ],
+          "type": "string",
           "default": "https://sourcegraph.com",
           "description": "The base URL of the Sourcegraph instance to use."
+        },
+        "sourcegraph.accessToken": {
+          "type": "string",
+          "description": "An access token for your account on the Sourcegraph instance configured in `sourcegraph.url`."
```

Comment:
We should make it clear that this is not required for regular use of the extension.